website/docs/dbt-cli/install/docker.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: "Docker"
+title: "Use Docker to install dbt"
+description: "You can use Docker to install dbt and adapter plugins from the command line."
 ---
 
-By v1.0.0, all adapter plugins maintained by dbt Labs will be available as official Docker images,
+By v1.0.0, dbt Core and all adapter plugins maintained by dbt Labs will be available as official Docker images,
 and available from a public registry.
 
```

Comment:
```suggestion
We recommend you use Docker to install in production because it includes dbt Core and all of its dependencies. You might might also use a Docker to install and develop locally if you don't have your python environment set up.
```